src/codelabs/iterables/index.md

```diff
@@ -27,24 +32,30 @@ Estimated time to complete this codelab: 60 minutes.
 
 ## Why do you need collections
 
-Collections are classes that represented a group of objects, also called "elements".
+Iterables are a kind of collection. Collections are classes that represented 
+a group of objects, also called "elements".
 
 A collection can be empty, or contain many elements in it. Depending on its purpose,
 a collection will follow a different data structure and have different implementations.
 These are some of the most common collection types:
 
-* [List][list class]: Used to access elements by their index.
```

Comment:
Actually... in the case where you're talking about specific collection types (not just iterables), "read" is too restrictive. "read and write" might be better... but maybe I should save that change for the copyedit.